docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -336,9 +337,17 @@ Scale-up of vicious cycle interventions (kangaroo care, breastfeeding promotion,
 
 * :ref:`Diarrheal Diseases Risk Effects <2019_risk_effect_diarrheal_diseases>`
 
+.. note::
+
+  :ref:`Suboptimal Breastfeeding Risk Effects (GBD 2020) <2020_risk_suboptimal_breastfeeding>` have been removed from this simulation.
```

Comment:
Because... NEBF impacts DD only until age 6 months, while our wasting model is relevant at age >6 month

Reply:
Ah, I see that you have included this reasoning below.